test/Test/HLint.hs

```diff
@@ -16,22 +17,11 @@ hlintSpec :: Spec
 hlintSpec = describe "HLint works on opened PRs" $ do
     it "works with non-code PRs" $
         pr1 >>= runLog . runHLint >>= shouldBe []
-    it "works on code PRs" $
-        pr2 >>= runLog . runHLint >>= shouldBe [etaReduce] . map show
     it "creates correct comment for PR 2" $
-        pr2 >>= runLog . runHLint >>= shouldBe [etaComment] . map createComment
+        pr2 >>= runLog . runHLint >>= shouldBe [etaComment] . map commentBody
```

Comment:
Let's also add a test for a hint that doesn't change the whole line, only a part :slightly_smiling_face: